scripts/stubsabot.py

```diff
@@ -320,12 +321,25 @@ async def suggest_typeshed_update(update: Update, session: aiohttp.ClientSession
             return
 
     body = "\n".join(f"{k}: {v}" for k, v in update.links.items())
-    body += """
 
-If stubtest fails for this PR:
-- Leave this PR open (as a reminder, and to prevent stubsabot from opening another PR)
-- Fix stubtest failures in another PR, then close this PR
-"""
+    stubtest_will_run = not meta.get("stubtest", {}).get("skip", False)
+    if stubtest_will_run:
+        body += textwrap.dedent(
+            """
+
+            If stubtest fails for this PR:
+            - Leave this PR open (as a reminder, and to prevent stubsabot from opening another PR)
+            - Fix stubtest failures in another PR, then close this PR
+            """
+        )
+    else:
+        body += textwrap.dedent(
+            f"""
+
+            :warning: Review this PR carefully, as stubtest is SKIPPED in CI for {update.distribution}! :warning:
```

Comment:
nit: doesn't need to be so forceful

```suggestion
            :warning: Review this PR manually, as stubtest is skipped in CI for {update.distribution}! :warning:
```